chore(acz): update existing milestones (#59)
logos-co/roadmap: content/vac/acz/rlnp2p/waku/rln-multi-epoch-constraints.md

### Justification

> rln-v3 should allow dynamic epoch sizes for users who may need smaller messaging needs, to optimize for stake used.

Reviewer: ```suggestion
Dynamic epoch sizes are required  for users who have smaller messaging needs, to optimize for stake used.
rln-v3 will allow that.
```
